pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -54,8 +54,10 @@ func setAccessLogAndBuildTCPFilter(env *model.Environment, node *model.Proxy, co
 		acc := &accesslog.AccessLog{
 			Name: xdsutil.FileAccessLog,
 		}
-		if util.IsXDSMarshalingToAnyEnabled(node) {
+		if util.IsProxyVersionGE11(node) {
 			buildAccessLog(fl, env)
+		}
+		if util.IsXDSMarshalingToAnyEnabled(node) {
 			acc.ConfigType = &accesslog.AccessLog_TypedConfig{TypedConfig: util.MessageToAny(fl)}
 		} else {
```

Comment:
same. I think something changed in the access log config that makes it not work for <1.1?

Reply:
IsXDSMarshalingToAnyEnabled Is checking for 11 as well.

Reply:
That method should be better named `shouldMarshalToAny()`

Reply:
@mandarjog it does check for 11 as well

```
// IsXDSMarshalingToAnyEnabled controls whether "marshaling to Any" feature is enabled.
func IsXDSMarshalingToAnyEnabled(node *model.Proxy) bool {
	return IsProxyVersionGE11(node) && !pilot.DisableXDSMarshalingToAny()
}
```